components/ui/ErrorContainer.tsx

```
@@ -1,5 +1,6 @@
 import { colors } from "@/constants";
 import { StyleSheet, View } from "react-native";
+import { StyleSheet, View } from "react-native";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'StyleSheet'. Consider to delete it or rename it.

'StyleSheet' is defined here:


(lint/suspicious/noRedeclare)